doc/bt/ZC_SET_LAYER.bt

```diff
@@ -0,0 +1,17 @@
+//------------------------------------------------
+//--- 010 Editor v8.0 Binary Template
+//
+//      File: ZC_SET_LAYER.bt
+//   Authors: celophi
+//   Version: i171140
+//   Purpose: Sets a UI overlay. This is used for cutscenes and possibly other cases.
```

Comment:
This sounds interesting. What exactly does it mean? Is this used for the phasing? Or is it just something related to the UI?

Reply:
I don't know what "phasing" means, but when you enter a cutscene, all of the UI elements disappear. The character cannot interact except with dialog controls. This is known as "director mode", I think, and this packet enables that.

Reply:
Phasing is dynamic instancing, you run into a part of the map and suddenly you see nothing but yourself and your party. In ToS that's used for "story" based cutscenes and battles. I was thinking the layer parameter might be like an instance/phase id. Though I guess it could be a padded byte toggle as well.

Reply:
Hm, doesn't look like a toggle
```text
12 0D FF FF FF FF FA 02 00 00
```